README.md

```diff
@@ -176,9 +176,21 @@ though this timeout will be suspended while the toast has focus or the mouse is
 - [Global attributes](https://html.spec.whatwg.org/multipage/dom.html#global-attributes)
 - `open`: a boolean attribute, determining whether the toast is visible or not (according to the default styles).
 By default toasts are not shown.
-- `type`: category of toast, conveying the semantic priority of the toast, and influencing its styling (both default and user-provided). This will correspond closely with certain [WAI-ARIA roles](https://w3c.github.io/using-aria/#aria-roles) 
-(more discussion in [#18](https://github.com/jackbsteinberg/std-toast/issues/18)).
-    - TODO([#18](https://github.com/jackbsteinberg/std-toast/issues/18)): decide on list of WAI-ARIA Roles to use, and decide on types to natively support, create, and style.
+- `type`: an [enumerated attribute](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#enumerated-attribute) giving the type of toast:
+  one of `"info"`, `"success"`, `"warning"`, or `"error"`.
+  This is used to convey the [semantics](https://html.spec.whatwg.org/multipage/dom.html#represents) of the toast,
```

Comment:
Thought: should the types be announced differently to screen readers?

For instance with this toast:

```html
<std-toast type="error" open>
  Access Denied!
</std-toast>
```

Should the screen reader read out something like `error: Access Denied!`? Or is this encompassed by the toasts' aria roles?

Reply:
My general understanding is that screen-reader users should not get content tailored only for them in this way, but I have to run for tonight and want to do more reading before making a definitive statement in that regard.